platform/android/java/editor/src/main/java/org/godotengine/editor/BaseGodotEditor.kt

```diff
@@ -194,19 +187,16 @@ abstract class BaseGodotEditor : GodotActivity() {
 	}
 
 	@CallSuper
-	protected open fun updateCommandLineParams(args: List<String>) {
+	protected override fun updateCommandLineParams(args: Array<String>) {
 		// Update the list of command line params with the new args
-		commandLineParams.clear()
-		if (args.isNotEmpty()) {
-			commandLineParams.addAll(args)
-		}
-		if (BuildConfig.BUILD_TYPE == "dev") {
-			commandLineParams.add("--benchmark")
+		val args = if (BuildConfig.BUILD_TYPE == "dev") {
+			arrayOf(*args, "--benchmark")
```

Comment:
nit: here you can just return `args + "--benchmark"`

Reply:
Perfect, loving kotlin more the more I learn of the syntax!